CuitService/Controllers/TaxInfoController.cs

```diff
@@ -1,52 +1,44 @@
+using Flurl.Http;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
 using System.Threading.Tasks;
+using Tavis.UriTemplates;
 
 namespace CuitService.Controllers
 {
     [Authorize]
     public class TaxInfoController
     {
         private readonly ILogger<TaxInfoController> _logger;
+        private readonly TaxInfoProviderConfiguration _cuitServiceConfiguration;
+        private readonly IFlurlClient _flurClient;
 
-        public TaxInfoController(ILogger<TaxInfoController> logger)
+        public TaxInfoController(ILogger<TaxInfoController> logger, IOptions<TaxInfoProviderConfiguration> cuitServiceConfiguration)
         {
             _logger = logger;
+            _cuitServiceConfiguration = cuitServiceConfiguration.Value;
+            _flurClient = new FlurlClient().AllowAnyHttpStatus();
```

Comment:
What do you think to inject IFluentClient in the constructor?

Reply:
I agree with @nreal that we should eventually improve this code, maybe removing _Flurl_ completely in favor of _HttpClient_.

But by the moment I think that current implementation is acceptable, or we could even simplify it a little more using [Fluent HTTP](https://flurl.dev/docs/fluent-http/) (without using `new`).

Internally Flurl has smart management of _HttpClient_ and thanks to _HttpTest_, we do not need to inject _IFlurlClient_ to test it.

Reply:
Why do you need to accept any HTTP status?

Reply:
Here you have an example without explicit _FlurlClient_ instances:

<img width="661" alt="image" src="https://user-images.githubusercontent.com/1157864/87609749-c398df00-c6d9-11ea-88c7-865f7e9330ce.png">

In my opinion, it is not so much flexible, but we do not need such flexibility (at least not yet)